### ISSUE TITLE
Enabler: Drupal NCIDS Footer Return to Top

### ISSUE THREAD

**Issue by olitharp-nci** [NCIOCPL/cgov-digital-platform#3509]
# Implement NCIDS Footer Return to Top
> **ESTIMATE** TBD

Need to implement changes to the footer made for the custom Return To Top
https://github.com/NCIOCPL/ncids/issues/374

Original analytics here
https://github.com/NCIOCPL/cgov-digital-platform/issues/3216

### Acceptance Criteria for Behaviors
*If a component changes between breakpoints, or things show/hide, that gets captured here* 

``` gherkin
 # todo
```

### Acceptance Criteria for Analytics
From #3216:
``` gherkin
Scenario: Back To Top link click
Given a user is on any page of cancer.gov
When the user clicks on the Back To Top link
Then there should be an NCIDataLayer Other event 
  And the event has the name "RightEdge:LinkClick"
  And the event has the the linkname "RightEdge:LinkClick"
  And this event should have the following data:
    | key      | value        |
    | linkText | Back To Top  |
    | location | Right Edge   |
    | section  | Back To Top  |
```

### Solution
* The the classname `usa-footer__return-to-top` gets replaced with `usa-footer__nci-return-to-top` on init, update required

**Comment by olitharp-nci** [comment]
https://github.com/NCIOCPL/ncids/issues/374 Text changes from “Return to Top” to “Back to Top”, do analytic requirements need to change?

**Comment by sarinapadilla** [comment]
Closing this ticket as it has been fixed by PR #3510!
